exporters/otlp/all/src/main/java/io/opentelemetry/exporter/otlp/trace/OtlpGrpcSpanExporter.java

```diff
@@ -20,9 +20,11 @@ public final class OtlpGrpcSpanExporter implements SpanExporter {
   private final GrpcExporter<TraceRequestMarshaler> delegate;
 
   /**
-   * Returns a new {@link OtlpGrpcSpanExporter} reading the configuration values from the
-   * environment and from system properties. System properties override values defined in the
-   * environment. If a configuration value is missing, it uses the default value.
```

Comment:
This same incorrect comment appears in `OtlpGrpcMetricExporter` and `OtlpGrpcLogRecordExporter`. Would be good to fix those as well.